### ISSUE TITLE
`oz send-tx` allows to call initialize method after contract being initialized

### ISSUE THREAD

**Issue by ylv-io** [OpenZeppelin/openzeppelin-sdk#1140]
![image](https://user-images.githubusercontent.com/3285000/61631761-da0d8580-ac93-11e9-9774-f64954146f6c.png)

**Comment by crazyrabbitLTC** [comment]
Isn't this expected behavior? Calling it a second time is akin to a normal revert, users should be allowed to call `initialize()` as many times as they want, the solidity code will enforce that it can be called only once.

**Comment by ylv-io** [comment]
Yes second call should fail. My point that if `initialized` is already called we should show the method in the list. Why show method which we know will fail?

**Comment by spalladino** [comment]
Closing as duplicate of #1035
